common.gypi

```diff
@@ -566,6 +573,42 @@
           'OPENSSL_NO_ASM',
         ],
       }],
+      ['OS == "zos"', {
+        'defines': [
+          '_XOPEN_SOURCE_EXTENDED',
```

Comment:
We should probably match the macros that we use in our wrapper:
https://github.ibm.com/open-z/node-build/blob/v16.x.zos/bin/njsc

For example, _ENHANCED_ASCII_EXT=0xFFFFFFFF should be set

Reply:
I added the missing macros from the wrapper